pgmpy/models/MarkovModel.py

```diff
@@ -177,10 +177,15 @@ def remove_factors(self, *factors):
         for factor in factors:
             self.factors.remove(factor)
 
-    def get_cardinality(self):
+    def get_cardinality(self,node=None):
         """
-        Returns a dictionary with the given factors as keys and their respective
-        cardinality as values.
+        Returns the cardinality of the node. If node is not specified returns
+        a dictionary with the given factors as keys and their respective cardinality
```

Comment:
@lohani2280 Shouldn't this be `given variable as keys` instead of `given factors as keys`.